app/templates/components/events/view/overview/event-setup-checklist.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="content">
+  <div class="header">{{t 'Event setup checklist'}}</div>
+</div>
+<div class="ui vertical steps">
```

Comment:
I have put the header in a content div and all the steps inside ui vertical steps in separate content div.